doxygen/src/pdsc_format.txt

```diff
@@ -295,13 +295,13 @@ assists searching for packages.
   </tr>
   <tr>
     <td>\ref element_keywords "keywords"</td>
-    <td>Defines keywords that might be used to find a \ref cp_SWComponents "Software Pack". Keywords may be also exported to web page to provide information for search engines, such as Google.</td>
+    <td>Defines keywords that might be used to find a software pack. Keywords may be also exported to web page to provide information for search engines, such as Google.</td>
     <td>group</td>
     <td>0..1 </td>
   </tr>
   <tr>
     <td>\ref element_generators "generators"</td>
-    <td>Specifies generator tools that have been used to create the \ref cp_SWComponents "Software Pack".</td>
+    <td>Specifies generator tools that have been used to create the software pack.</td>
```

Comment:
This does not sound right. This is the classic generator which generate a gpdsc (not pdsc) that is included into the project. What am I missing? Not sure when this got introduced.

Reply:
Reohrased.